mongofiles/main/mongofiles.go

```diff
@@ -39,6 +40,13 @@ func main() {
 	mf, err := mongofiles.New(opts)
 	if err != nil {
 		log.Logv(log.Always, err.Error())
+		if setupErr, ok := err.(mongofiles.SetupError); ok {
+			if setupErr.Code == util.ExitBadOptions {
+				log.Logvf(log.Always, "try mongofiles --help for more information")
```

Comment:
We're ignoring the actual error from SetupError in this case. Can we make the log msg include setupErr.Error()?

Reply:
we're still logging the error message on line 42, even if the error is a `SetupError`. The additional checking is to see if we should tell the user to use `--help` or not before trying again.

Reply:
ah missed that - resolving.